client/src/pages/reports/Show.tsx

```diff
@@ -323,10 +324,18 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }: ReportShowProps) => {
   const [showCustomFields, setShowCustomFields] = useState(
     !!Settings.fields.report.customFields
   )
+  const [attachments, setAttachments] = useState([])
   const { uuid } = useParams()
   const { loading, error, data, refetch } = API.useApiQuery(GQL_GET_REPORT, {
     uuid
   })
+
+  useEffect(() => {
+    if (data?.report) {
+      const report = new Report(data.report)
+      setAttachments(report.attachments || [])
+    }
+  }, [data])
```

Comment:
Why not simply:
```javascript
  useEffect(() => {
    setAttachments(data?.report?.attachments || [])
  }, [data])
```
I would also suggest to put the `useEffect` after `usePageTitle`.